### ISSUE TITLE
[Error] TypeError: validate_object_path() argument 1 must be string, not None

### ISSUE THREAD

**Issue by Wurzelkoch** [quodlibet/quodlibet#2759]
* What did you try to do when the error occurred?
Error occurs at every try to start quodlibet. I cannot remember what I changed since the last time when quodlibet was doing well. Occurs in 3.7.1-2 and the below mentioned version.

Error Details:
```
Traceback (most recent call last):

  File "/usr/bin/quodlibet", line 14, in <module>
    sys.exit(main())

  File "/usr/lib/python2.7/dist-packages/quodlibet/main.py", line 168, in main
    session.init("quodlibet")

  File "/usr/lib/python2.7/dist-packages/quodlibet/qltk/session.py", line 29, in init
    client = bus.get_object("org.gnome.SessionManager", client_path)

  File "/usr/lib/python2.7/dist-packages/dbus/bus.py", line 241, in get_object
    follow_name_owner_changes=follow_name_owner_changes)

  File "/usr/lib/python2.7/dist-packages/dbus/proxies.py", line 244, in __init__
    _dbus_bindings.validate_object_path(object_path)

TypeError: validate_object_path() argument 1 must be string, not None


Version: 3.9.1
Python: 2.7.13 (default, Nov 24 2017, 17:33:09) 
[GCC 6.3.0 20170516]
Platform: Linux-4.9.0-4-amd64-x86_64-with-debian-9.3
```

**Comment by lazka** [comment]
Thanks

**Comment by lazka** [comment]
Are you using LXDE or any non-GNOME desktop environment?

**Comment by Wurzelkoch** [comment]
You're right, I'm using LXDE.

**Comment by lazka** [comment]
See #2435 fixed in 4.0

We have a debian repo if needed: https://quodlibet.readthedocs.io/en/latest/downloads.html#debian
